backend/WSS_Site/settings.py

```diff
@@ -244,3 +244,10 @@
 local_settings_path = os.path.join(os.path.dirname(__file__), 'local_settings.py')
 if os.path.exists(local_settings_path):
     exec(open(local_settings_path, 'rb').read())
+
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_USE_TLS = True
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_PORT = 587
+EMAIL_HOST_USER = 'TODO@EMRAN.MERCI'
```

Comment:
This is not a secret and can be hardcoded.

Reply:
Indeed